content/rdi/quickstart/write-behind-guide.md

```diff
@@ -1,7 +1,7 @@
 ---
-Title: Write-behind quickstart
-linkTitle: Write-behind
-description: Get started creating a write-behind pipeline
+Title: Write-behind & Read-Through quickstart
+linkTitle: Write-behind & Read-Through
+description: Get started creating a write-behind and read-through pipeline
```

Comment:
I think we should have one term that covers this, as we will add write-through at some point. It can be Redis keyspace, or something similar.

Reply:
pipeline for redis synchronization with a downstream database (write behind, read through, write through (pending)) ?

Reply:
done